src/config.rs

```diff
@@ -37,6 +37,35 @@ impl Default for SortOrder {
     }
 }
 
+#[derive(Debug, Eq, PartialEq, Hash, Copy, Clone)]
+#[derive(Serialize, Deserialize)]
+#[serde(deny_unknown_fields)]
+pub enum SassOutputStyle {
+    Nested,
+    Expanded,
+    Compact,
+    Compressed,
+}
+
+#[derive(Debug, PartialEq)]
+#[derive(Serialize, Deserialize)]
+#[serde(deny_unknown_fields)]
+pub struct SassOptions {
+    pub sass_dir: String,
+    pub extensions: Vec<String>,
+    pub style: SassOutputStyle,
+}
+
+impl Default for SassOptions {
+    fn default() -> SassOptions {
+        SassOptions {
+            sass_dir: "_sass".to_owned(),
+            extensions: vec!["scss".to_owned(), "sass".to_owned()],
```

Comment:
Since I'm new to sass, could you explain the difference in the extensions?

Reply:
I am not super sure in this either, but from my understand sass is the old (pre version 3.0) version of the syntax and scss (standing for "Sassy CSS", which is also a Superset of CSS) now is the standard. I have seen older or long-term-maintained projects still use the old extension, though I am not even sure they are compatible or the current libsass-version does indeed support (or even detect) the old style.

Though without certainty on this, there is the argument to be made that this shouldn't include both, I suppose 🤔 .

Reply:
I defer to you to decide (or anyone else who knows better about these things).  Like above, we can always add support for it later if needed.